vxvas2nets/vas2nets_sms.py

```diff
@@ -167,9 +206,12 @@ def handle_outbound_message(self, message):
         if missing_fields:
             returnValue((yield self.reject_message(message, missing_fields)))
 
-        # TODO status event for request timeout
-        # TODO status event for succcessful requests
-        resp = yield self.send_message(message)
+        try:
+            resp = yield self.send_message(message)
+        except (ResponseNeverReceived, ConnectingCancelledError,
+                CancelledError):
```

Comment:
I'm not wild about catching all of these, but I think they can all be raised and there isn't a common base class. Maybe it would be better to catch `Exception`, but that seems ugly too.